pytorch/inference/docker/1.11/py3/cu115/Dockerfile.gpu

```diff
@@ -164,12 +164,12 @@ RUN curl -L -o ~/mambaforge.sh https://github.com/conda-forge/miniforge/releases
     # Below 2 are included in miniconda base, but not mamba so need to install
     conda-content-trust \
     charset-normalizer \
- && /opt/conda/bin/conda install -c pytorch magma-cuda115=2.6.1 
+ && /opt/conda/bin/conda install -c pytorch magma-cuda115=2.6.1 \
 # Upstream conda looks to have moved to 4.13 which is incompatible with mamba 0.22.1 and will fail the conda-forge installs.
 # having "conda update conda" before the "conda -c conda-forge" commands will automatically update conda to 4.13.
 # Moving conda update conda" after the "conda -c conda-forge" commands keep conda at 4.12 but will update other packages using
 # the current conda 4.12
-RUN /opt/conda/bin/conda update -y conda \
+ && /opt/conda/bin/conda update -y conda \
```

Comment:
Why are we doing this? Just to avoid a new step?

Reply:
New RUN command would create another image layer. It’s best to avoid doing that, as the clean command won’t be able to reduce the image size, because the clean step is at the next layer.